app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::ReservationsController < ApplicationController
+  def index
```

Comment:
- It looks like this controller could be used for anybody without some kind of restriction, I invite you to add the `before_action` command on `reservations_controllers.rb` to be used only for logged users. 🐱